services/self-service/src/main/java/com/epam/dlab/backendapi/domain/EndpointDTO.java

```diff
@@ -23,14 +23,22 @@
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.Data;
+import org.hibernate.validator.constraints.NotEmpty;
+import org.hibernate.validator.constraints.URL;
 
 
 @Data
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class EndpointDTO {
-
+	/*
+	 the URL_REGEXP_VALIDATION constant is a template for URL pattern,
+	  i.e for url verification like "https://localhost:8084/a/$-*^.oLeh;/"
+	 */
+	private static final String URL_REGEXP_VALIDATION = "^(http(s)?)://[-a-zA-Z0-9+&@#/%?=~_|!:,.;]*[-a-zA-Z0-9+&@#/%=~_|]";
+	private static final String URL_RESPONSE_MESSAGE = "is empty or contains improper format, symbols ";
```

Comment:
I wonder we can change the message to "is in the improper format". Because **is empty** is a particular case of **improper format**. How do you think?

Reply:
Yeap, a common message looks better.
What do you think, if we apply the javax Pattern annotation to validate the Name field?